src/components/editor/index.tsx

```diff
@@ -75,6 +75,14 @@ const Editor: Component<Props> = (props) => {
       },
     });
 
+    editor.onKeyDown(e => {
+      if ((e.metaKey || e.ctrlKey) && e.browserEvent.key === 's') {
```

Comment:
This will also match <kbd>Control + S</kbd> on macOS but:

1. I see another example of this pattern elsewhere in the codebase
2. I don't see any helpers/examples for differentiating between Windows/Mac